package.json

```diff
@@ -26,11 +26,12 @@
     "request": "^2.69.0"
   },
   "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
Old test were done with [should](https://github.com/bandwidthcom/node-bandwidth/pull/53/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R43) If we're going to move to chai & `expect` we **_should_** talk about that to make sure we're down with updating all the tests.

Also @nbragdon had mentioned reevaluating test framework